content/en/core/releases/3.15.0.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "3.15.0 release notes"
+linkTitle: "3.15.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+3.15.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+None.
+
+## Highlights
+
+### Support reporting of Bikram Sambat dates using text forms
```

Comment:
```suggestion
### Support reporting of Bikram Sambat dates using JSON forms
```

"text" is such an overloaded term that I was confused by what it meant the first time I read this.  I don't know if there is a more proper term for these forms, but I see them called "JSON forms" on the linked doc page...

Reply:
Yes sure. It may also be my kiwi terminology - over here we use "text" instead of "SMS", because we like ambiguity I guess?